<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>imageSelected</class>
 <widget class="QMainWindow" name="imageSelected">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>667</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IRdetection</string>
  </property>
  <property name="windowIcon">
   <iconset resource="appressources.qrc">
    <normaloff>:/Images/logo_VISirX.png</normaloff>:/Images/logo_VISirX.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSpinBox" name="numOfClusters">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>280</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>42</width>
      <height>0</height>
     </size>
    </property>
    <property name="minimum">
     <number>2</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
   </widget>
   <widget class="QLabel" name="clusters">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Clusters K</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>270</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>K-means</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelcopyImage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>211</width>
      <height>181</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labeltraitementImage">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>191</width>
      <height>181</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="contour">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>270</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Contours</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelContour">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>201</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="LbelsizeContour">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>279</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>667</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSegmenter">
   <property name="text">
    <string>segmenter</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="appressources.qrc"/>
 </resources>
 <connections/>
</ui>
